client/components/shortcode/frame.jsx

```diff
@@ -122,6 +123,19 @@ export default React.createClass( {
 		this.props.onLoad( event );
 	},
 
+	getSandboxAttribute() {
+		// Tags of shortcodes that do not work if rendered in a sandboxed iframe.
+		const iframeNoSandbox = [
+			'wpvideo'
+		]
+		const shortcodeTag = shortcodeUtils.parse( this.props.shortcode ).tag;
+		const trustedHost = iframeNoSandbox.some( function ( trustedShortcodeTag ) {
```

Comment:
What leads you to conclude that `wpvideo` shortcodes are more trustworthy than any other shortcode?

The original concern being addressed here is that the REST API shortcode endpoint is [handled in the Jetpack plugin](https://github.com/Automattic/jetpack/blob/master/json-endpoints/class.wpcom-json-api-render-shortcode-endpoint.php), so there's a worry about the HTML being returned having originated from an external host.